benchmarks/ALIA/sv-benchmarks/2019.array-cav/array_init_var_plus_ind2.sl

```diff
@@ -0,0 +1,34 @@
+(set-logic ALIA)
+
+(synth-inv inv_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int) (k Int)))
+
+(declare-primed-var a (Array Int Int))
+(declare-primed-var j Int)
+(declare-primed-var i Int)
+(declare-primed-var n Int)
+(declare-primed-var x Int)
+(declare-primed-var k Int)
+
+
+(define-fun pre_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int) (k Int)) Bool
+  (and (>= n 0) (= i 0) (= j 0) (= k 0)))
+
+(define-fun trans_fun ((a (Array Int Int)) (n Int) (j Int) (i Int) (x Int) (k Int) (a! (Array Int Int)) (n! Int) (j! Int) (i! Int) (x! Int) (k! Int)) Bool
+  (and (and (< i n) (= x 0))
+       (= n! n)
+       (= k! (- k 1))
+       (= j! (+ j 1))
```

Comment:
Is this the source file: https://github.com/sosy-lab/sv-benchmarks/blob/master/c/array-cav19/array_init_var_plus_ind2.c?

In the source, `k` and `j` change by `i` not `1`!